triaging.rst

```diff
@@ -9,6 +9,41 @@ and developers.
 Contributors with the Developer role on the `issue tracker`_ can triage issues
 directly without any assistance.
 
+Additionally, this section provides an overview of the bug triager role.
+
+Bug Triager
+------------
+
+The bug triager is a role dedicated towards improving workflow efficiency and
+lessening the workload of the core developers. The expectations of this role
+expand upon the "python triager" role on the `issue tracker`_. The responsibilities
+listed below are primarily centered around the Python GitHub repositories. This
+extends beyond cpython, to the other repos such as devguide, core-workflow, and
```

Comment:
Since current bug triagers are only triaging CPython repo, I'm unsure if we want to grant triage role to the other repos, or just CPython for now. I've asked in https://github.com/python/core-workflow/issues/334#issuecomment-513282372 for opinion.

Reply:
Personally, I've of the opinion that it should include any `Python core` repository, especially devguide and core-workflow. I expanded upon this a bit on the [issue page](https://github.com/python/core-workflow/issues/334).